libs/base/Data/Strings.idr

```diff
@@ -16,11 +16,16 @@ partial
 foldl1 : (a -> a -> a) -> List a -> a
 foldl1 f (x::xs) = foldl f x xs
 
+%foreign
+  "scheme:string-unpack"
+export
+fastUnpack : String -> List Char
+
```

Comment:
I think we could use a comment explaining when / why to pick
`fastUnpack` over `unpack`.

Reply:
Thanks, I added one.